wpilibc/src/main/native/include/frc/ADIS16470_IMU.h

```diff
@@ -223,6 +223,13 @@ class ADIS16470_IMU : public wpi::Sendable,
    */
   units::degree_t GetAngle(IMUAxis axis) const;
 
+  /**
+   * Returns the Z axis angle (CCW positive).
+   *
+   * @return The Z axis angle (CCW positive).
+   */
+  units::degree_t GetAngle() const;
```

Comment:
Any reason to not add a default for the `axis` parameter instead of adding an overload?

Reply:
It would certainly be more idiomatic that way.